src/lib/components/molecules/first-past-the-post-waffle/stories.module.scss

```diff
@@ -0,0 +1,25 @@
+.line {
+    height: 100%;
+    width: 1px;
+    left: calc(50% - 0.5px);
+    background-color: var(--primary-text-color);
+    font-size: 14px;
+    margin-top: 0;
+    position: absolute;
+    bottom: 3px;
+    z-index: 2;
```

Comment:
Should this styling not be done in the CSS for the FPTPWaffle? Otherwise it becomes less useful as a component